client/views/msg_action.tpl

```diff
@@ -1,6 +1,6 @@
 <div class="msg {{type}}{{#if self}} self{{/if}}{{#if highlight}} highlight{{/if}}"
 	data-type="{{type}}" id="msg-{{id}}" data-time="{{time}}">
-	<span class="time tooltipped tooltipped-e" aria-label="{{localetime time}}">
+	<span class="time" tooltipped tooltipped-e" title="{{localetime time}}">
```

Comment:
That's because it is. Groan, I'll take a look later. I will be focusing on the modularising and my other PRs over the condensing one just now. I will pick it up after that.